conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: numpy
+  version: "" # Set at build time.
+
+source:
+  git_url: ..
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - openblas    # [linux]
+    - python
+    - cython
+    - gcc         # [unix]
+
+  run:
+    - openblas    # [linux]
+    - python
+    - libgcc      # [unix]
+
+test:
```

Comment:
For running tests:

  test:
    requires:
      nose
      gfortran (I guess, didn't check the name - for f2py tests)